networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/gossip/topics/BlockTopicHandler.java

```diff
@@ -21,14 +21,18 @@
 import tech.pegasys.artemis.networking.eth2.gossip.events.GossipedBlockEvent;
 import tech.pegasys.artemis.networking.eth2.gossip.topics.validation.BlockValidationResult;
 import tech.pegasys.artemis.networking.eth2.gossip.topics.validation.BlockValidator;
+import tech.pegasys.artemis.storage.client.RecentChainData;
 
 public class BlockTopicHandler extends Eth2TopicHandler<SignedBeaconBlock> {
-  public static final String BLOCKS_TOPIC = "/eth2/beacon_block/ssz";
+  public static String TOPIC_NAME = "beacon_block";
   private final EventBus eventBus;
   private final BlockValidator blockValidator;
 
-  public BlockTopicHandler(final EventBus eventBus, final BlockValidator blockValidator) {
-    super(eventBus);
+  public BlockTopicHandler(
+      final EventBus eventBus,
+      final BlockValidator blockValidator,
+      final RecentChainData recentChainData) {
```

Comment:
I'd pass the `ForkDigest` into the topic handler rather than passing in `recentChainData`.  We'll need to be able to explicitly create topics for specific forks.

Reply:
Done.